examples/resources/nobl9_direct_logic_monitor/resource.tf

```
@@ -0,0 +1,14 @@
+resource "nobl9_direct_logic_monitor" "logic_monitor" {
+  name = "logic_monitor"
+  project = "logic_monitor"
```

```suggestion
  project = "logic-monitor"
```
This won't work with the underscore.